microsoft_cve_rag/application/etl/extractor.py

```
@@ -2,16 +2,18 @@
-from typing import Any, Dict, Optional, List, Tuple
-from application.services.document_service import DocumentService
-from pymongo import ASCENDING, DESCENDING
-from bson import ObjectId
-# import uuid
-from datetime import datetime, timedelta, timezone
 import hashlib
 import json
 import logging
 
+# import uuid
+from datetime import datetime, timedelta, timezone
```

Ruff (0.8.2)

`datetime.timedelta` imported but unused

Remove unused import: `datetime.timedelta`

(F401)